src/de/mod.rs

```diff
@@ -1,6 +1,1324 @@
+//! </div>
+//! <div style="background:rgba(80, 240, 100, 0.20);padding:0.75em;">
+//!
+//! Merging of the text / CDATA content is tracked in the issue [#474] and
+//! will be available in the next release.
+//! </div>
+//! <!-- TODO: document an error type returned -->
+//! </td>
+//! </tr>
+//! <!-- 2 ===================================================================================== -->
+//! <tr>
+//! <td>
+//!
+//! Content of attributes and text / CDATA content of elements (including mixed
+//! text and CDATA content), which represents a space-delimited lists, as
+//! specified in the XML Schema specification for [`xs:list`] `simpleType`:
+//!
+//! ```xml
+//! <... ...="element1 element2 ..." />
+//! ```
+//! ```xml
+//! <...>
+//!   element1
+//!   element2
+//!   ...
+//! </...>
+//! ```
+//! ```xml
+//! <...><![CDATA[
+//!   element1
+//!   element2
+//!   ...
+//! ]]></...>
+//! ```
+//! [`xs:list`]: https://www.w3.org/TR/xmlschema11-2/#list-datatypes
+//! </td>
+//! <td>
+//!
+//! Use any type that deserialized using [`deserialize_seq()`] call, for example:
+//!
+//! ```
+//! // FIXME: #474, merging mixed text / CDATA
+//! // content does not work yet
+//! type List = Vec<u32>;
+//! ```
+//!
+//! See the next row to learn where in your struct definition you should
+//! use that type.
+//!
+//! According to the XML Schema specification, delimiters for elements is one
+//! or more space (`' '`, `'\r'`, `'\n'`, and `'\t'`) character(s).
+//!
+//! <div style="background:rgba(120,145,255,0.45);padding:0.75em;">
+//!
+//! NOTE: according to the XML Schema restrictions, you cannot escape those
+//! white-space characters, so list elements are _always_ will not contain them.
+//! In practice you will usually use `xs:list`s for lists of numbers or enumerated
+//! values which looks like identifiers in many languages, for example, `item`,
+//! `some_item` or `some-item`, so that shouldn't be a problem.
+//!
+//! NOTE: according to the XML Schema specification, list elements can be
+//! delimited only by spaces. Other delimiters (for example, commas) are not
+//! allowed.
+//!
+//! </div>
+//! <div style="background:rgba(80, 240, 100, 0.20);padding:0.75em;">
+//!
+//! Merging of the text / CDATA content is tracked in the issue [#474] and
+//! will be available in the next release.
+//! </div>
+//!
+//! [`deserialize_seq()`]: de::Deserializer::deserialize_seq
+//! </td>
+//! </tr>
+//! <!-- 3 ===================================================================================== -->
+//! <tr>
+//! <td>
+//! A typical XML with attributes. The root tag name does not matter:
+//!
+//! ```xml
+//! <any-tag one="..." two="..."/>
+//! ```
+//! </td>
+//! <td>
+//!
+//! A structure where an each XML attribute are mapped to a field with a name
+//! started from `@`. Because an `@` character is not allowed in the Rust identifier,
+//! you should use `#[serde(rename = "@...")]` attribute to rename it.
+//! Name of the struct itself does not matter:
```

Comment:
```
//! A structure where each XML attribute is mapped to a field with a name
//! starting with `@`. Because Rust identifiers do not permit the `@` character,
//! you should use the `#[serde(rename = "@...")]` attribute to rename it.
//! The name of the struct itself does not matter:
```

Reply:
Done